src/utils/builder.ts

```diff
@@ -375,13 +399,13 @@ export function setActionNodeElm(pathway: Pathway, nodeKey: string, elm: ElmLibr
   });
 }
 
-export function getNodeType(node?: ActionNode | BranchNode | PathwayNode | null): string {
-  if (!node || node.nodeTypeIsUndefined) {
+export function getNodeType(
```

Comment:
I know this method was here before your PR but this method is better suited for `utils/nodeUtils` than this file